.ci/Makefile

```diff
@@ -23,6 +23,9 @@ NOT_USED := $(shell test -e $(ROOT_DIR)/deployer-config.yml && sed "s;roleId:.*;
 endif
 endif
 
+# are we running any e2e tests in this build against SNAPSHOT versions of the stack?
+SNAPSHOT_TEST_TARGET ?= $(findstring -SNAPSHOT,$(STACK_VERSION))
```

Comment:
It works but I would try not to add another env var and keep the Makefile simple.